waspc/data/Generator/templates/react-app/src/auth/useAuth.ts

```diff
@@ -23,5 +26,8 @@ export async function getMe(): Promise<User | null> {
   }
 }
 
-getMe.queryCacheKey = ['auth/me']
-getMe.route = { method: HttpMethod.Get, path: '/auth/me' }
+addMetadataToQuery(getMe, {
+  relativeQueryPath: getMePath,
```

Comment:
Shouldn't the `relativeQueryPath` not begin with the `/` (for use in the `queryCacheKey`?

Reply:
I mean I guess it's not a strict requirement, but for consistency with ops?

Reply:
Change it 👍